scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/sop.yaml

```diff
@@ -0,0 +1,49 @@
+FileHeader:
+  library_Suffix: 'SO'
+  device_type: 'SOP'
+
+SOP-8-1EP_4.57x4.57mm_P1.27mm_EP4.57x4.45mm:
+  size_source: 'https://ww2.minicircuits.com/case_style/XX112.pdf'
+  body_size_x:
+    nominal: 4.57
+    tolerance: 0.01
```

Comment:
I think the PL number takes the imperial number as the reference. (It would not make sense that the tolerance changes depending on which measurement system one uses.)

Meaning i am really convinced that 0.127 is the correct tolerance.